coffee-backend/src/main/kotlin/chat/letscoffee/security/oauth2/OAuth2AuthenticationFailureHandler.kt

```diff
@@ -1,29 +1,29 @@
 package chat.letscoffee.security.oauth2
 
+import chat.letscoffee.security.oauth2.HttpCookieOAuth2AuthorizationRequestRepository.Companion.REDIRECT_URI_PARAM_COOKIE_NAME
 import chat.letscoffee.util.CookieUtils
-import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.security.core.AuthenticationException
 import org.springframework.security.web.authentication.SimpleUrlAuthenticationFailureHandler
 import org.springframework.stereotype.Component
 import org.springframework.web.util.UriComponentsBuilder
 import java.io.IOException
 import javax.servlet.ServletException
-import javax.servlet.http.Cookie
 import javax.servlet.http.HttpServletRequest
 import javax.servlet.http.HttpServletResponse
 
 @Component
-class OAuth2AuthenticationFailureHandler(private val httpCookieOAuth2AuthorizationRequestRepository: HttpCookieOAuth2AuthorizationRequestRepository) : SimpleUrlAuthenticationFailureHandler() {
+class OAuth2AuthenticationFailureHandler(
+    private val httpCookieOAuth2AuthorizationRequestRepository: HttpCookieOAuth2AuthorizationRequestRepository
```

Comment:
So Short lines 😭

Reply:
Don't worry, I added 100 spaces at the end of the line, we just can't see it becaue it's invisible 🙈